.travis.yml

```diff
@@ -8,6 +8,6 @@ env:
 language: node_js
 
 node_js:
-  - "5"
+  - "6"
```

Comment:
while `latest` is a nice terminology and maybe better than just "node", I agree `stable` is ambigous, include LTS versions to the options I would wonder for it.

If there's a chance for breaking changes, link `stable` to any stable release and do not update it anymore. It won't break anything and you'll have your deprecated alias.